src/API/src/allDataFileBuilder.service.ts

```diff
@@ -0,0 +1,56 @@
+const fs = require('fs');
+import { Inject, Injectable } from '@nestjs/common';
+import { handleLastIngestLock } from 'src/ingest/utils/triggerCsvRebuild';
```

Comment:
Can you change imports such as this to local imports? So this would be './ingest/utils/...'